receiver/vcenterreceiver/testdata/metrics/expected.yaml

```diff
@@ -1,6 +1,9 @@
 resourceMetrics:
   - resource:
       attributes:
+        - key: vcenter.datacenter.name
```

Comment:
Since it'll be optional by default, I'd expect this file to be unchanged. However, it would be great to have another test which enables it, so we can see that it's producing expected values.